diff --git a/content/browser/accessibility/accessibility_event_recorder_mac.mm b/content/browser/accessibility/accessibility_event_recorder_mac.mm
index c6864118d007da39a16b821c0351322f67065fbb..f5d56e0ea072005a16067df2a78f41b0d5943c60 100644
--- a/content/browser/accessibility/accessibility_event_recorder_mac.mm
+++ b/content/browser/accessibility/accessibility_event_recorder_mac.mm
@@ -131,7 +131,8 @@ void AccessibilityEventRecorderMac::EventReceived(AXUIElementRef element,
                                                   CFDictionaryRef user_info) {
   std::string notification_str = base::SysCFStringRefToUTF8(notification);
 
-  auto formatter = AccessibilityTreeFormatterMac();
+  //auto formatter = AccessibilityTreeFormatterMac();
+  AccessibilityTreeFormatterMac formatter;
   formatter.SetPropertyFilters(property_filters_,
                                ui::AXTreeFormatter::kFiltersDefaultSet);
 
